mailpoet/mailpoet
# Feat/let users authorise their email addresses in the plugin - MAILPOET-4300

[MAILPOET-4300](https://mailpoet.atlassian.net/browse/MAILPOET-4300)

Please start reviewing from https://github.com/mailpoet/mailpoet/commit/4198bade172d6214f7360149eb02d52163358710

Depends on https://github.com/mailpoet/mailpoet/pull/4216

Testing:
* Test anywhere with links to MP Shop Email authorization page e.g. `/authorization?email=NEW_EMAIL`
* Confirm you can add a new email
* Confirm you get the confirmation email
* Confirm after clicking on the confirmation email and waiting for some seconds, the modal window update with new messages
* Confirm you can use the new email for sending newsletters
